commands.c

```diff
@@ -830,18 +830,18 @@ void mutt_print_message(struct Mailbox *m, struct EmailList *el)
 /**
  * mutt_select_sort - Ask the user for a sort method
  * @param reverse If true make it a reverse sort
- * @retval num Sort type, see #SortType
+ * @retval true The sort type changed
  */
 bool mutt_select_sort(bool reverse)
 {
   enum SortType sort = SORT_DATE;
 
   switch (mutt_multi_choice(reverse ?
-                                /* L10N: The highlighted letters must match the "Sort" options */
+                                /* L10N: The highlighted letters must match the "Rev-Sort" options */
                                 _("Rev-Sort "
                                   "(d)ate,(f)rm,(r)ecv,(s)ubj,t(o),(t)hread,(u)"
                                   "nsort,si(z)e,s(c)ore,s(p)am,(l)abel?") :
-                                /* L10N: The highlighted letters must match the "Rev-Sort" options */
+                                /* L10N: The highlighted letters must match the "Sort" options */
```

Comment:
Actually, these were correct.
What they're saying is:
The highlighted letters in the string **below**, should match those in the string **mentioned** in the comment.

`L10N` comments are copied to the `.po` files where the order of the strings may not match the code.

Reply:
okay, that makes more sense. I've removed this hunk of the change in my latest forced push.